### DISCUSSION TITLE
site is not secure without www in the url

### DISCUSSION THREAD

**Discussion by infinitegame111** [community/community#51760] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

I am making a website for my game and when loading https://viper101.org there is a smart screen error but when loading https://www.viper101.org it loads just fine. Can someone help?

**Comment by DataJuggler** [comment]
Without knowing your hosting provider or SSL Certificate Issuer, it is kind of hard to answer.

If you are using IIS, you need to edit the binding of your the direct url (the one without the www) and ensure the SSL certificate is selected.

![image](https://user-images.githubusercontent.com/26331133/229366603-50a24826-9755-4d17-9b83-c34bb91099b2.png)

Then select the SSL certificate.

![image](https://user-images.githubusercontent.com/26331133/229366679-ea88cd20-5ef9-42ab-8ebf-6613eec6f330.png)
